src/test/java/org/openrewrite/java/testing/mockito/VerifyZeroToNoMoreInteractionsTest.java

```diff
@@ -0,0 +1,235 @@
+/*
+ * Copyright 2024 the original author or authors.
+ * <p>
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * <p>
+ * https://www.apache.org/licenses/LICENSE-2.0
+ * <p>
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.openrewrite.java.testing.mockito;
+
+import org.intellij.lang.annotations.Language;
+import org.junit.jupiter.api.Test;
+import org.openrewrite.DocumentExample;
+import org.openrewrite.InMemoryExecutionContext;
+import org.openrewrite.java.JavaParser;
+import org.openrewrite.test.RecipeSpec;
+import org.openrewrite.test.RewriteTest;
+
+import static org.openrewrite.java.Assertions.java;
+import static org.openrewrite.maven.Assertions.pomXml;
+
+class VerifyZeroToNoMoreInteractionsTest implements RewriteTest {
+
+    @Language("xml")
+    private static final String POM_XML_WITH_MOCKITO_2 = """
+      <project>
+        <modelVersion>4.0.0</modelVersion>
+        <groupId>bla.bla</groupId>
+        <artifactId>bla-bla</artifactId>
+        <version>1.0.0</version>
+        <dependencies>
+          <dependency>
+              <groupId>org.mockito</groupId>
+              <artifactId>mockito-core</artifactId>
+              <version>2.17.0</version>
+              <scope>test</scope>
+          </dependency>
+        </dependencies>
+      </project>
+      """;
+
+    @Override
+    public void defaults(RecipeSpec spec) {
+        spec
+          .parser(JavaParser.fromJavaVersion()
+            .classpathFromResources(new InMemoryExecutionContext(), "mockito-core", "mockito-junit-jupiter", "junit-jupiter-api"))
+          .recipe(new VerifyZeroToNoMoreInteractions());
+    }
+
+    @Test
+    @DocumentExample
+    void shouldReplaceToNoMoreInteractions() {
+        //language=java
+        rewriteRun(
+          pomXml(POM_XML_WITH_MOCKITO_2),
+          java(
+                """
+            import org.junit.jupiter.api.extension.ExtendWith;
+            import org.mockito.junit.jupiter.MockitoExtension;
+            import org.junit.jupiter.api.Test;
+
+            import static org.mockito.Mockito.verifyZeroInteractions;
+
+            @ExtendWith(MockitoExtension.class)
+            class MyTest {
+                @Test
+                void test() {
+                    verifyZeroInteractions(System.out);
+                }
+            }
+            """, """
```

Comment:
Try to place the start of the next text block on a new line too; the bot doesn't yet enforce that, but probably should.